domain/schema_validator_test.go

```diff
@@ -30,7 +30,7 @@ type leaseGrantItem struct {
 func (*testSuite) TestSchemaValidator(c *C) {
 	defer testleak.AfterTest(c)()
 
-	lease := 5 * time.Millisecond
+	lease := 10 * time.Millisecond
```

Comment:
Why change the lease's value? We still sleep a lease time at line63.

Reply:
I am also confused by this. May you should add a description for this PR about why do you increase lease time.

Reply:
I think the useful changes is in ine113.

Reply:
> We still sleep a lease time at line63
Yes, I found nothing wrong with the test logic

So I change the lease's value to make it tolerant to call `time.Now()` cost and channel receive operation.

Reply:
Make sense

Reply:
When the lease is short, and get lease cost too long, the lease mechanism can't work.

```
leaseGrantTime  = time.Now()
<- leaseItem
```

By the time receiver get the lease, it's already stale.

Does this explain make sense? @zhexuany